<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>faturas</class>
 <widget class="QWidget" name="faturas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Faturas</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Id atual:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblID">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnVoltar">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Voltar</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>541</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Pesquisar por:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxInput">
        <item>
         <property name="text">
          <string>Nenhuma descrição</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fornecedor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Valor</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Data de vencimento</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Status de pagamento</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnPesquisar">
        <property name="text">
         <string>Pesquisar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="edtPesquisar"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Fornecedor</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Valor</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Data de vencimento</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status de pagamento</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Data de vencimento:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Fornecedor:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtFornecedor">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Valor:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtValor">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_7"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Id</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edtId">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Status de pagamento:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbxStatus">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <item>
           <property name="text">
            <string>Em pendente</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pagado</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="btnLimparTudo">
          <property name="text">
           <string>Limpar tudo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnAdicionar">
          <property name="text">
           <string>Adicionar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnPagar">
          <property name="text">
           <string>Pagar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEditar">
          <property name="text">
           <string>Editar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
